luci-app-adblock-fast/htdocs/luci-static/resources/view/adblock-fast/overview.js

```diff
@@ -16,8 +16,8 @@ var pkg = {
 	get URL() {
 		return "https://docs.openwrt.melmac.net/" + pkg.Name + "/";
 	},
-	humanFileSize: function (bytes, si = false, dp = 2) {
-		return `%${si ? 1000 : 1024}.${dp ?? 0}mB`.format(bytes);
+	humanFileSize: function (bytes) {
```

Comment:
I'd like to keep an old function to allow using different options in the future.

Reply:
It needs for mintance and adds complexity. It would be easy ti implement if needed. Still not sure why to show mibibytes as megabytes in si